keras/layers/preprocessing/category_encoding.py

```diff
@@ -90,7 +90,7 @@ class CategoryEncoding(base_layer.Layer):
         inputs to the layer must integers in the range `0 <= value <
         num_tokens`, or an error will be thrown.
       output_mode: Specification for the output of the layer.
-        Defaults to `"multi_hot"`. Values can be `"one_hot"`, `"multi_hot"` or
+        Values can be `"one_hot"`, `"multi_hot"` or
```

Comment:
I think we should remove the double quote here, and just make it `one_hot`. Same for all the changes in this PR.

Reply:
`Union["one_hot"`, "multi_hot","count"]` is no longer the type?

Reply:
`Union["one_hot", "multi_hot","count"]` is no longer the type?